### PR TITLE
no_public_key_note

### PR DESCRIPTION
**Pull request by Zoh63392187**
Added note for "No public Key" for Unkown accounts
image

### CODE REVIEW

**Comment by shefass** [comment]
Personally I don't like this message, it creates image that you must to
"activate" your acc and without "activating" it, it is not active. Maybe
better: to use all Burst features please activate it.
Or maybe drop this activation word. Only a opinion.

2019-11-17, sk, 15:55 Zoh <notifications@github.com> rašė:

**Comment by blankey1337** [comment]
There seems to be two colors after `Note` -  should should only be one `Note:` vs `Note::`

**Comment by Zoh63392187** [comment]
Note:: fixed.
Added Ajax call instead of link.

File: html/ui/js/brs.js [hunk 1]
```diff
@@ -550,6 +550,38 @@ var BRS = (function(BRS, $, undefined) {
         }
     };
 
+	BRS.activateAccount = function activate_acc(){
+		$.ajax({
+			type: 'GET',
+			url: 'https://explorer.burstcoin.network/api_activate.php?pkey=' + BRS.publicKey + '&acc=' + BRS.accountRS ,
```
Comment (ohager) [inline html/ui/js/brs.js]: IMHO the host should be configurable...

File: html/ui/js/brs.js [hunk 2]
```diff
@@ -550,6 +550,38 @@ var BRS = (function(BRS, $, undefined) {
...
+				} else {
+					$.notify($.t("no_public_key_warning_note_fail") + "<br>" + response.response_text, {
+					type: 'danger',
+						offset: {
+							x: 5,
+							y: 60
+							}
+					});
+				}
+				console.log(response);
```
Comment (ohager) [inline html/ui/js/brs.js]: console.log :roll_eyes:

File: html/ui/locales/bg.json [hunk 3]
```diff
@@ -558,6 +558,7 @@
    "no_products_found" : "Няма намерени продукти.",
    "no_products_listed_yet" : "Все още не сте вписали продукти за продажба",
    "no_public_key_warning" : "Предупреждение!: вашата сметка няма публичен ключ! Това означава, че не е защитена като други сметки. Трябва да направите изходящ превод да решите проблема.",
+   "no_public_key_warning_note": "<br><strong>Note:<\/strong> This account does not have a public key! Activate your account <a id='act_lnk' href='#' onclick='BRS.activateAccount();return false;'>here</a>",
```
Comment (ohager) [inline html/ui/locales/bg.json]: I use this for translation: 

https://electronjs.org/apps/i18n-manager

And embedding logic into translations is kind of weird

File: html/ui/js/brs.js (hunk 2 above)
Comment (Zoh63392187) [reply html/ui/js/brs.js]: Ups :)

File: html/ui/locales/bg.json (hunk 3 above)
Comment (Zoh63392187) [reply html/ui/locales/bg.json]: Removed

File: html/ui/js/brs.js (hunk 1 above)
Comment (Zoh63392187) [reply html/ui/js/brs.js]: Will try and add it to brs-default.properties

Comment (Zoh63392187) [reply html/ui/js/brs.js]: Implemented in settings and PR updated

File: html/ui/js/brs.settings.js [hunk 4]
```diff
@@ -2,6 +2,8 @@
  * @depends {brs.js}
  */
 var BRS = (function(BRS, $, undefined) {
+	var API_URL = "https://explorer.burstcoin.network/api_activate.php";
```
Comment (harry1453) [inline html/ui/js/brs.settings.js]: This is my main problem with this, it adds a dependency on an external service, which I don't consider appropriate for BRS. I wouldn't feel the same way about Phoenix, but I think BRS must remain fully independent

Comment (blankey1337) [reply html/ui/js/brs.settings.js:5]: I tend to agree. Further, we should feature freeze this UI, as in no new features. Closing this.

**Comment by blankey1337** [comment]
Closed as legacy UI is feature frozen.

**Pull request closed without merging**